### DISCUSSION TITLE
Accessibility issues with screen readers when using the copilot extension

### DISCUSSION THREAD

**Discussion by jeanbrazcosta** [community/community#15705] [Copilot Conversations] [answered]
Hello, after being approved to use the copilote extension, I noticed that the NVDA screen reader does not read the suggestions given by the extension.

I have Windows 10 latest version of vscode and latest version of NVDA reader screens.

Do you think it would be possible to improve accessibility for those who use screen readers?

**Comment by RafaelFernandesBR** [comment]
you have to ctrl+enter to display the suggestions

**Comment by RafaelFernandesBR** [comment, marked as answer]
you have to ctrl+enter to display the suggestions

**Comment by jeanbrazcosta** [comment]
When I press control +enter I don't get any suggestions and vscode breaks
lines in my code.

Em qui., 12 de mai. de 2022 às 19:28, RafaelFernandesBR <
***@***.***> escreveu:

**Comment by jeanbrazcosta** [comment]
@RafaelFernandesBR a feedback that I'm not getting from the extension when using a screen reader is a suggestion of a line of code.

example: I started creating the following class:

public class student {
String name;
Int age;
when typing another public I heard the suggestion sound and when pressing tab a constructor was automatically created for the class in question.

But I only had the audio feedback ie before pressing tab, I don't know what suggestion the copilot is giving me.

I use Windows with NVDA screen reader and the latest version of vscode.

**Reply by RafaelFernandesBR** [reply]
@jeanbrazcosta
I tested here, really, can't tell the suggestion before pressing the tab.
I didn't even know there was this function

**Comment by Yeldece** [comment]
Indeed needs an improvement

**Comment by hamidrezaabroshan** [comment]
I have this problem too.
I'm using the latest version of vscode and Nvda screen reader.
As you may know, Nvda is free and open source. So, you can test it with vscode.
I suggest that use a mechanism like Vscode IntelliSense 
It is fully accessible.
But it needs some difference.
IntelliSense  is activated and pups up by default but, if Copilot does have a suggestion, it should play a sound as it is and by pressing a key it pups up.
Thanks.

**Comment by jeanbrazcosta** [comment]
It is worth mentioning that, after listening to the copilot suggestion
sound, if I'm not mistaken control + ] shows another suggestion, and this
mechanism is also not read by nvda.

Also, I forgot to say that when I see the ten code suggestions pressing
control+enter, I can't stop on any and press the accept button, I have to
copy the code and insert it into my class. It would be interesting if the
accept button was focused by the cursor when the user presses tab or some
other shortcut key.

Em qua., 8 de jun. de 2022 às 11:56, hamidrezaabroshan <
***@***.***> escreveu:
